Embedding widgets/widget-customization/README.md

```diff
@@ -5,40 +5,40 @@ languages:
 - javascript
 products:
 - azure-video-indexer
-description: "Azure Video Analyzer for Media widgets"
+description: "Azure Video Indexer widgets"
 ---
 
-#  Azure Video Analyzer for Media widgets
+#  Azure Video Indexer widgets
 
 [![License: MIT](https://img.shields.io/badge/License-MIT-yellow.svg)](https://opensource.org/licenses/MIT)
 [![TypeScript](https://img.shields.io/badge/%3C%2F%3E-TypeScript-%230074c1.svg)](https://www.typescriptlang.org/)
 [![code style: prettier](https://img.shields.io/badge/code_style-prettier-f8bc45.svg)](https://github.com/prettier/prettier)
 
 ![avam_widgets_banner_github.png](https://user-images.githubusercontent.com/51399662/135491488-10ec3d7a-e573-4ddf-898a-337412a63150.gif)
 
-This repo contains the Azure Video Analyzer for Media widgets package. Below you can find documentation and examples on how to use these pieces.
+This repo contains the Azure Video Indexer widgets package. Below you can find documentation and examples on how to use these pieces.
 
 ## Introduction
 
 [Video Analyzer for Media](https://docs.microsoft.com/en-us/azure/azure-video-analyzer/video-analyzer-for-media-docs) (formerly Video Indexer) is a cloud application, part of Azure Applied AI Services, built on Azure Media Services and Azure Cognitive Services (such as the Face, Translator, Computer Vision, and Speech). It enables you to extract the insights from your videos using Video Analyzer for Media video and audio models.
```

Comment:
Video Indexer? Maybe run find "Video analyzer for media" (without azure)

Reply:
good catch !